client/cypress/integration/edit-lesson.spec.ts

```diff
@@ -58,6 +64,7 @@ describe("lesson screen", () => {
       "have",
       "Question the student needs to answer, e.g. 'What are the colors in RGB?'"
     );
+    cy.get("#image").should("have", "");
```

Comment:
when I started upgrading to cypress `5.0` from `5.3`, the biggest thing it complains about is this use of `have` (which is not an assertion).

Can you try to use something else instead, I think would be like

```js
cy.get("#image").should("have.attr", "src", "");
```